b2sdk/b2http.py

```diff
@@ -22,8 +22,8 @@
 from typing import Any
 
 import requests
-from requests.adapters import HTTPAdapter
 
+from b2sdk.urllib3._connection import HTTPAdapterWithContinue
```

Comment:
I got scared a little that we are switching from requests to raw urllib, but apparently `b2sdk.urllib3` module name is just wrong here